build.sbt

```diff
@@ -37,6 +37,9 @@ ThisBuild / libraryDependencies ++= Seq(
   scalaTest % Test,
   mockito % Test)
 
+Test / javaOptions += "-Xmx2G"
```

Comment:
Why are we testing with 2GB? Is it for the Documentation? In this way we may not detect memory problems, right?

Reply:
The point is to set a value that will be used in any platform, so the tests are run in a similar environment. If you want to lower it, it's ok, but check if it works. The problem usually is the documentation tests anyway.

Reply:
It seems ok to me!